scootapi/locate.go

```diff
@@ -17,7 +17,8 @@ import (
 
 // Get the path of the file containing the address for scootapi to use
 func GetScootapiAddrPath() string {
-	return path.Join(os.Getenv("HOME"), ".cloudscootaddr")
+	optionalId := os.Getenv("SCOOT_ID") // Used to connect to a different set of scoot processes.
```

Comment:
This is a knob enabling devs to run concurrent tests against different cloudExec instances. I don't see a need to make this more prominent. Would it suffice to add a comment "Allows devs to concurrently test different scoot instances"?